lib/cartopy/mpl/slippy_image_artist.py

```diff
@@ -47,7 +47,12 @@ def draw(self, renderer, *args, **kwargs):
         if not self.get_visible():
             return
 
-        ax = self.get_axes()
+        try:
+            # matplotlib 1.5.0+
+            ax = self.axes
```

Comment:
Does this actually cause an attribute error? Looks like get_axes has always pointed to the axes attribute: https://github.com/matplotlib/matplotlib/blob/v1.3.1rc2/lib/matplotlib/artist.py#L179